wp-google-login.php

```diff
@@ -16,7 +16,7 @@
 define( 'WP_GOOGLE_LOGIN_PATH', untrailingslashit( plugin_dir_path( __FILE__ ) ) );
 define( 'WP_GOOGLE_LOGIN_URL', untrailingslashit( plugin_dir_url( __FILE__ ) ) );
 define( 'WP_GOOGLE_LOGIN_PLUGIN_NAME', plugin_basename( __FILE__ ) );
-define( 'WP_GOOGLE_LOGIN_VERSION', '0.1' );
+define( 'WP_GOOGLE_LOGIN_VERSION', '1.0' );
```

Comment:
Add a stable version in README.md for user understanding.

Reply:
Added